queries.sql

```diff
@@ -1,31 +1,150 @@
-/*Queries that provide answers to the questions from all projects.*/
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = TRUE;
+SELECT * FROM animals WHERE name <> 'Gabumon';
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+-- Update the species column to 'unspecified'
```

Comment:
- [ ] I noticed that in your `queries.sql` file, the right queries for this Milestone were not included in your file. As a result, all your screenshots are also not pertaining to this milestone. So kindly refer to the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md) and include the right queries for this Milestone and add screenshots of the results of the queries in your Pull Request Summary :wink: 

**Project Requirements**

![Screenshot from 2023-07-28 11-40-29](https://github.com/kayjeee/kayjeeSQL/assets/102699958/078da757-5a80-4a15-bac5-7d7ee2f95254)

Reply:
done

Reply:
done